session4-6/personalwebsite/src/layouts/default.hbs

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>{{title}}</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+  <style>
+    .carousel {
+      margin: auto;
+      max-width: 1100px;
+      
+    }
+    html,body {
+  margin:0px;
+  padding:0px;
+  font-family: cursive;
+  }
+ul {
+    background: #f5f5f5;
+   }   
+li {
+    list-style: none;
+    display: inline-block;
+    padding:10px;    
+}
+li a {
+    text-decoration: none;
+}
+li a:hover{
+	text-decoration: underline;
+}
+section {
+width:100%;
+clear:both;
+padding: 20px 0px;
+}
+aside,section {padding: 0px 20px;}
+article {padding: 20px;}
+a {
+    color: chocolate;
+}
+
+  </style>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
```

Comment:
Use local script files and concatenate them together into single vendor.min.js using gulp